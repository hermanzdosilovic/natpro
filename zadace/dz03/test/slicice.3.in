8 3 4
56
100
999
10002
53214
100001
777778
999996
