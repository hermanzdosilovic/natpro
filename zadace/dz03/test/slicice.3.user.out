slavko
slavko
mirko
mirko
slavko
mirko
mirko
mirko
